docs/src/pages/components/cdk-components/text-input/api.jsx

```diff
@@ -119,16 +119,19 @@ const textInputPropsTable = () => {
           entered is not valid) will be passed to this function. An example of
           this object is: {"{ "}
           <code>value: value, error: error</code>
-          {" }"}. If there is no error, error will be null.
+          {" }"}. If there is no error, error will not be defined.
         </td>
       </tr>
       <tr>
         <td>error: string</td>
```

Comment:
`pattern`, `minLength` and `maxLength` prop descriptions need to be updated too (_"... the error parameter of both events will be null"_).